SmartDeviceLink/SDLRadioControlData.h

```diff
@@ -0,0 +1,93 @@
+//
+//  SDLRadioControlData.h
+//
+
+#import "SDLRPCMessage.h"
+#import "SDLRadioBand.h"
+#import "SDLRadioState.h"
+
+@class SDLRdsData;
+
+/**
+ * Include information (both read-only and changeable data) about a remote control radio module.
+ */
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLRadioControlData : SDLRPCStruct
+
+- (instancetype)initWithFrequencyInteger:(nullable NSNumber<SDLInt> *)frequencyInteger frequencyFraction:(nullable NSNumber<SDLInt> *)frequencyFraction band:(nullable SDLRadioBand)band rdsData:(nullable SDLRdsData *)rdsData availableHDs:(nullable NSNumber<SDLInt> *)availableHDs hdChannel:(nullable NSNumber<SDLInt> *)hdChannel signalStrength:(nullable NSNumber<SDLInt> *)signalStrength signalChangeThreshold:(nullable NSNumber<SDLInt> *)signalChangeThreshold radioEnable:(nullable NSNumber<SDLBool> *)radioEnable state:(nullable SDLRadioState)state;
+
+/**
+ * @abstract The integer part of the frequency ie for 101.7 this value should be 101
+ *
+ * Integer
+ */
+@property (nullable, strong, nonatomic) NSNumber<SDLInt> *frequencyInteger;
+
+/**
+ * @abstract The fractional part of the frequency for 101.7 is 7
+ *
+ * Integer
+ */
+@property (nullable, strong, nonatomic) NSNumber<SDLInt> *frequencyFraction;
+
+/**
+ * @abstract Radio band value
+ *
+ * SDLRadioBand
+ */
+@property (nullable, strong, nonatomic) SDLRadioBand band;
+
+/**
+ *@abstract Read only parameter. See RdsData data type for details.
+ *
+ *SDLRdsData
+ */
+@property (nullable, strong, nonatomic) SDLRdsData *rdsData;
```

Comment:
A number of these properties are readonly, and the documentation should note it is such.